spring-boot/admin/src/main/resources/application.yml

```diff
@@ -107,3 +126,12 @@ security:
       userInfoUri: https://api.github.com/user
 
 management.security.enabled: false
+
+# Proxy configuration files are stored in this directory. The value of this parameter needs to direct to the same
+# folder as spring.cloud.config.server.native.searchLocations from the Config Servers application.yml.
+proxy.config.files.folder: ${user.dir}/.temp-gateway-config-file-storage
```

Comment:
I am wondering: if I start-up admin and config server from different folders, wouldn't `user.dir` be different? On the other hand, this is maybe only relevant for local dev and thus not a problem?!

Another option would be to use a folder like `${user.home}/.coatrack/temp-gateway-config-file-storage`, which is independent of the folder from which admin/config-server have been started.

Reply:
Ah sry, I forgot to mention that. I already tried out the `user.home` directory but this did not work out well. I think the reason is that the home is not equally defined in different environments. iirc, then using `user.home` did not work correctly in the CI pipeline (the CI runner to be precise) and failed.